packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -420,11 +421,14 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
     }, [currentAppView]);
 
     /*
-     * React on changes in the venue prop.
+     * React on changes in the venue prop. If not defined, check if it is defined in app config.
      */
     useEffect(() => {
-        setCurrentVenueName(venue);
-    }, [venue]);
+        const desiredVenue = venue ?? appConfig?.appSettings?.venue ?? null;
+        if (!isNullOrUndefined(desiredVenue)) {
+            setCurrentVenueName(desiredVenue);
+        }
```

Comment:
Can it be shortened to this?

```suggestion
        setCurrentVenueName(venue ?? appConfig?.appSettings?.venue);
```

Setting it to undefined should be alright, is my cautious assumption.